js-module-base/pom.xml

```diff
@@ -80,12 +82,12 @@
                         <version>1.8</version>
                         <executions>
                             <execution>
-                                <phase>initialize</phase>
+                                <phase>package</phase>
```

Comment:
Is this the actual fix?

Reply:
One of them ... there were multiple issues.